lib/sail/value_cast.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require 'time'
```

Comment:
I'd say let's use DateTime. This require will no longer be needed.

Reply:
@vinistock Done